lib/image-diff.js

```diff
@@ -73,7 +73,7 @@ ImageDiff.createDiff = function (options, cb) {
     // DEV: According to http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=17284
     // DEV: These values are the total square root mean square (RMSE) pixel difference across all pixels and its percentage
     // TODO: This is not very multi-lengual =(
-    var resultInfo = stderr.match(/all: (\d+\.?\d*) \((\d+\.?\d*)\)/);
+    var resultInfo = stderr.match(/all: (\d+\.?\d*) \((.*?)\)/);
```

Comment:
I remember why I changed this - in case of only a few pixel difference, we get a result like this from compare.exe:
  Channel distortion: RMSE
    red: 0.00129297 (5.07046e-006)
    green: 0 (0)
    blue: 0.00158356 (6.21002e-006)
    all: 0.00118031 (4.62868e-006)
So, we can either use \(.*\) or /\d+\.?\d*(e-\d+)?/ to allow for something small